Assets/Scripts/Audio/AudioController.cs

```diff
@@ -92,7 +92,10 @@ int musicVolume
 	// Backup channels that are dynamically created to allow SFX to finish without being cut off (empty when no SFX are playing)
 	List<AudioSource> tempSFXChannels = new List<AudioSource>();
 
-	[Header("Sweeteners")]
+    // A HashSet of all active (currently playing) SFX
+    HashSet<AudioClip> activeSFX = new HashSet<AudioClip>();
```

Comment:
This looks good! The only note to add is that you should also remove clips from the `HashSet` if the `Stop` method is called inside of `AudioController`